pkg/config/config_template.yaml

```diff
@@ -387,6 +392,13 @@ api_key:
   ## @param obfuscation - object - optional
   ## Defines obfuscation rules for sensitive data. Disabled by default.
   ## See https://docs.datadoghq.com/tracing/guide/agent-obfuscation
+  #
+  # obfuscation:
+  #   <SERVICE_NAME>:
```

Comment:
I think you misunderstood this. It's not an arbitrary service name. We currently have settings for `mongodb`, `elasticsearch`, `redis`, `memcached`, `http`, `remove_stack_traces`. We should also keep the smaller comments which describe what each sub-option means. Because it doesn't make sense on its own.

Reply:
The description of this parameter is too big, this is why I created a dedicated Guide for in in the Public documentation, the PR is still open I'm currently integrating your feedback: https://github.com/DataDog/documentation/pull/4551